src/files/json.ts

```diff
@@ -22,9 +22,14 @@ async function readJSONText(file: BIDSFile): Promise<string> {
 
 export async function loadJSON(file: BIDSFile): Promise<Record<string, unknown>> {
   const text = await readJSONText(file) // Raise encoding errors
+  let parsedText;
   try {
-    return JSON.parse(text)
+    parsedText = JSON.parse(text)
   } catch (error) {
     throw { key: 'JSON_INVALID' } // Raise syntax errors
   }
+  if (Array.isArray(parsedText) || typeof parsedText !== "object") {
+    throw { key: 'JSON_NOT_AN_OBJECT' }
```

Comment:
Would be good to show what we found. I think we use `evidence`?

```suggestion
    throw { key: 'JSON_NOT_AN_OBJECT', evidence: text.substring(0, 10) + (text.length > 10 ? '...' : '') }
```

Reply:
Personally I'd still end up cat'ing the file if I received the error. But no harm in returning evidence.  `decoder.decode` used by readJSONText should always return a string so should be safe there.